testing/endtoend/helpers/helpers.go

```diff
@@ -132,7 +132,7 @@ func WaitForTextInFile(src *os.File, match string) error {
 
 	select {
 	case <-ctx.Done():
-		return fmt.Errorf("could not find requested text \"%s\" in %s before deadline:\n", match, f.Name())
```

Comment:
why change this?

Reply:
new line is not needed at the end. this is not a log